### PR TITLE
feat: removing GIT dependency in create command

### PR DESCRIPTION
**Pull request by medjedovicm**
## Issue

#1211 

## Intent

The initial issue was that when the `create` command was run on the machine that had no `GIT` installed, the CLI was failing without handling such a case.

We decided it is a better option to remove GIT completely, and to download the repo and `docs` submodule, manually.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
image
- [x] Any new functionality has been unit tested.
- [ ] All unit tests are passing (`npm test`).
- [ ] All CI checks are green.
- [x] JSDoc comments have been added or updated.
- [x] Reviewer is assigned.

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -75,57 +87,76 @@ export async function createTemplateApp(folderPath: string, template: string) {
...
+      folderPath,
+      `https://github.com/sasjs/template_${template}.git`,
+      'https://github.com/sasjs/docs'
+    )
     return resolve()
   })
 }
 
+/**
+ * It will download and unzip `sasjs/docs` into newly created folder into `docs` subfolder
```
Comment (allanbowe) [inline src/utils/utils.ts:100]: This function will first download / unzip the target app from `repoUrl`, then download / unzip the `sasjs/docs` repo into a subdirectory named 'docs'.  

Finally, if `installDependencies` is `true`, the `npm install` command will be executed.

**Comment by VladislavParhomchik** [comment]
Mocked tests failed:
image

**Review by YuryShkoda — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "copy:doxy": "copyfiles -u 2 src/doxy/* build/doxy/",
     "test": "npm run test:mocked && npm run test:server",
     "test:server": "jest --silent --runInBand --config=jest.server.config.js",
-    "test:mocked": "jest --silent --runInBand --config=jest.config.js --coverage",
+    "test:mocked": "jest --runInBand --config=jest.config.js --coverage",
```
Comment (YuryShkoda) [inline package.json:17]: why `silent` has been removed?

File: package.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@sasjs/adapter": "3.9.4",
+    "@sasjs/adapter": "3.10.3",
```
Comment (YuryShkoda) [inline package.json:50]: there is a newer version of `@sasjs\adapter@3.10.8`

Comment (allanbowe) [reply package.json]: bumped

File: src/utils/utils.ts [hunk 4]
```diff
@@ -40,95 +41,169 @@ export function diff(a: any[], b: any[]) {
...
+ * @param folderPath full path of the directory where desired seed app will be created
+ * @param repoUrl sasjs/{seed-app} repo url
+ * @param docsUrl sasjs/docs repo url
+ * @params installDependencies whether or not to do `npm install`
+ */
 function createApp(
   folderPath: string,
   repoUrl: string,
+  docsUrl: string,
+  errorCallback: (err: string) => void,
```
Comment (YuryShkoda) [inline src/utils/utils.ts:120]: `errorCallback` is the same for all app, so `createApp` can just reject promise with error by default

**Review by YuryShkoda — DISMISSED** [review]

**Comment by allanbowe** [comment]
also closes #1212

File: src/utils/utils.ts [hunk 5]
```diff
@@ -40,95 +41,169 @@ export function diff(a: any[], b: any[]) {
...
+  shelljs.cp('-r', `./*-main/.`, docsFolderPath)
+  shelljs.rm('-rf', [`./*-main`])
+  shelljs.rm('-rf', [`./main.zip`])
+}
+
+function downloadFile(url: string): ShellString {
+  if (isWindows()) {
+    return shelljs.exec(`curl ${url} -LO -f`, { silent: true })
+  } else {
+    return shelljs.exec(`wget ${url}`, { silent: true })
```
Comment (YuryShkoda) [inline src/utils/utils.ts:204]: Not all Mac OS has `wget`, but the `curl` should be available across all platforms.

**Comment by medjedovicm** [comment]
For tests to pass, this must be merged and bumped in the CLI.

https://github.com/sasjs/utils/pull/197

**Pull request merged**